document_page/views/document_page.xml

```diff
@@ -117,9 +117,9 @@
                 <field name="create_uid"/>
                 <field name="content_uid"/>
                 <group expand="0" string="Group By...">
-                    <filter string="Category" context="{'group_by':'parent_id'}"/>
-                    <filter string="Author" context="{'group_by':'create_uid'}"/>
-                    <filter string="Last Contributor" context="{'group_by':'content_uid'}"/>
+                    <filter name="category" string="Category" context="{'group_by':'parent_id'}"/>
```

Comment:
You can call it `group_by_category` for distinguishing it from other filters over the same field (and the rest applies the same)